storage/src/client.rs

```diff
@@ -222,6 +222,7 @@ mod test {
         let ts = DefaultTokenSourceProvider::new(Config {
             audience: None,
             scopes: Some(&SCOPES),
+            sub: None
```

Comment:
You will also need to modify [google-cloud-default](https://github.com/yoshidan/google-cloud-rust/blob/f1cd195a745ffa6f07dc7b7d368792a28e787dda/default/src/lib.rs#L36).